packages/core/src/mixins/primitives/reatomArray.test.ts

```diff
@@ -0,0 +1,58 @@
+import { describe, test, expect } from 'vitest'
```

Comment:
Please, do it for all tests

```suggestion
import { describe, test, expect } from 'test'
```